packages/edit-site/src/components/style-book/categories.ts

```diff
@@ -1,6 +1,7 @@
 /**
  * WordPress dependencies
  */
+// @ts-ignore
```

Comment:
We usually prefer `@ts-expect-error` because, when the underlying TS error gets fixed, it will complain and remind you to remove the comment.

Also, it's usually a good idea to add a comment about why we're ignoring / expecting an error

Reply:
Thanks! Updated.